portality/templates/publisher/preservation.html

```diff
@@ -0,0 +1,117 @@
+{% extends "publisher/publisher_base.html" %}
+
+{% block page_title %}Upload article XML{% endblock %}
+
+{% block publisher_content %}
+
+<div class="row">
+  <header class="col-md-4">
+    <h2>Upload preservation file</h2>
+
+    <div class="alert">
+      <ul>
+        <li>Know all about preservation <a href="/preservation">here</a></li>
+        <li>Check help documentation for creating preservation package <a href="{{ url_for('publisher.help') }}#preservation">here</a></li>
+        <li> <a href="/static/doaj/preservation_example.zip">Download </a>example zip file</li>
+      </ul>
+    </div>
+  </header>
+
+  <form id="upload_form" class="col-md-8 form form--compact" method="POST" action="/publisher/preservation" enctype="multipart/form-data">
+    <fieldset>
+      <h3 class="form__header">Preservation File</h3>
+      <div class="form__question">
+        <label for="upload-xml-file">Select a file</label>
+        <input type="file" id="upload-xml-file" name="file">
+        <p><small>Must be less than 50Mb.</small></p>
+      </div>
+    </fieldset>
+    <button type="submit" id="preservation" class="button button--primary">Upload</button>
+  </form>
+</div>
+
+<hr>
+
+{% if previous|length > 0 %}
+<h2>History of uploads (showing last {{previous|length}})</h2>
+
+<table id="previous_files" style="width: 100%; table-layout:fixed" class="tablesorter">
+    <thead>
+        <tr>
+            <th style="width: 17%">Upload Date</th>
+            <th style="width: 20%">Filename</th>
+            <th style="width: 15%">Upload Status</th>
+            <th style="width: 36%">Notes</th>
+        </tr>
+    </thead>
+{% for file in previous %}
+    <tr>
+        <td>{{ file.created_timestamp | utc_timestamp("%d %b %Y %H:%M:%S %Z") }}</td>
+        <td style="word-wrap: break-word">
+            {{file.filename}}
+        </td>
+        <td>
+            {% if file.status == "initiated" %}
+                <span style="color: #000066">initiated</span>
```

Comment:
Inline styles should be moved to scss files and colours should be the constants from _palette.scss file